.test/tests/ghost-basics/run.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+serverImage="$1"
+
+# Use a client image with curl for testing
+clientImage='buildpack-deps:buster-curl'
+# ensure the clientImage is ready and available
+if ! docker image inspect "$clientImage" &> /dev/null; then
+	docker pull "$clientImage" > /dev/null
+fi
+
+mysqlImage='mysql:8.0'
+# ensure the mysqlImage is ready and available
+if ! docker image inspect "$mysqlImage" &> /dev/null; then
+	docker pull "$mysqlImage" > /dev/null
+fi
+mysqlUser="user-$RANDOM"
+mysqlPassword="password-$RANDOM"
+mysqlDatabase="database-$RANDOM"
+
+# Create an instance of the container-under-test
+mysqlCid="$(
+	docker run -d \
+		-e MYSQL_RANDOM_ROOT_PASSWORD=1 \
+		-e MYSQL_USER="$mysqlUser" \
+		-e MYSQL_PASSWORD="$mysqlPassword" \
+		-e MYSQL_DATABASE="$mysqlDatabase" \
+		"$mysqlImage"
+)"
+trap "docker rm -vf $mysqlCid > /dev/null" EXIT
+
+# "Unknown database error" / "ECONNREFUSED" (and Ghost just crashing hard)
+. "$dir/../../retry.sh" --tries 30 'docker exec -i -e MYSQL_PWD="$mysqlPassword" "$mysqlCid" mysql -h127.0.0.1 -u"$mysqlUser" --silent "$mysqlDatabase" <<<"SELECT 1"'
```

Comment:
>```
>/home/runner/work/ghost/ghost/.test/tests/ghost-basics/run.sh: line 36: /home/runner/work/ghost/ghost/.test/tests/ghost-basics/../../retry.sh: No such file or directory
>	'ghost-basics' [4/4]...failed
>```

🤔 what do we do about these "lib" scripts?

Reply:
:sob: